opentelemetry-sdk/src/opentelemetry/sdk/_metrics/__init__.py

```diff
@@ -84,34 +117,54 @@ class MeterProvider(APIMeterProvider):
 
     def __init__(
         self,
+        *,
+        metric_readers: Sequence[MetricReader],
         resource: Resource = Resource.create({}),
+        views: Sequence[View] = (),
         shutdown_on_exit: bool = True,
     ):
         self._resource = resource
+        self._views = views
         self._atexit_handler = None
 
         if shutdown_on_exit:
             self._atexit_handler = register(self.shutdown)
 
-        self._metric_readers = []
+        self._metric_readers = metric_readers
         self._metric_exporters = []
```

Comment:
How come the `_metric_exporters` exist in the `MeterProvider`? I know it specifies `Shutdown MUST be implemented at least by invoking Shutdown on all registered MetricReader and MetricExporter instances.` in the spec but shouldn't the `MetricsReader` handle the calls and exporters are "registered" onto individual MetricReaders?